src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -164,11 +164,24 @@ private static bool ApplyChangesFromTC33(JObject sarifLog)
 
                     // https://github.com/oasis-tcs/sarif-spec/issues/302
                     MoveAllStackFrameAddressesToLocation(run);
+
+                    // https://github.com/oasis-tcs/sarif-spec/issues/352
+                    RenameMessageIdToId(run);
                 }
             }
             return true;
         }
 
+        private static void RenameMessageIdToId(JObject run)
+        {
+            var universallyRenamedMembers = new Dictionary<string, string>
+            {
+                ["messageId"] = "id",
+            };
+
+            RecursivePropertyRename(run, universallyRenamedMembers);
```

Comment:
>RecursivePropertyRename(run, universallyRenamedMembers); [](start = 12, length = 56)

hmm.. i can see why we chose this method for renames, there are simply way too many routes to message object in our current schema. I was trying to make a list of all paths last evening and it was ridiculous, but using this method instead didnt occur to me.